### PR TITLE
Calvin/artist inbox n requests

### PR DESCRIPTION
**Pull request by calvind777**
## Feature Name
Feature Description
artist inbox react component, controller to create requests

missing: authentication, but waiting on cancan in general

### Screenshots

CC: @clairelin135

### CODE REVIEW

**Comment by calvind777** [comment]
also modified requests table to contain a buyerid, artistid, and workid, eliminating the roundabout way i did it before

**Comment by calvind777** [comment]
can't create a button yet because our permissioning for the currently logged in buyer doesn't work yet (via cancan)

**Comment by calvind777** [comment]
but i added the preliminary works controller where the detailed Work view will live in like "/works/1" will show detailed info, the pic, and so on for that work

File: app/controllers/api/artists_controller.rb [hunk 1]
```diff
@@ -30,4 +30,24 @@ def works
     end
   end
 
+  def requests
+    artist = Artist.find(params[:id])
+    requests = artist.requests
+    requests_full = []
+    requests.each do |request|
+      puts request.buyer
+      puts request.work
```
Comment (clairelin135) [inline app/controllers/api/artists_controller.rb]: Remove puts statement

File: app/controllers/api/works_controller.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def create
   def update
     work = Work.find(params[:id])
     new_work = work.update(params)
-    render_json_message(:ok, message: 'Request successfully updated!')
+    render_json_message(:ok, message: 'Work successfully updated!')
```
Comment (clairelin135) [inline app/controllers/api/works_controller.rb]: Thanks for making these fixes! 🎉 🎉

File: app/views/works/show.html.slim [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.container.dashboard-container
+  = react_component "detailed_work", {work: @work}
```
Comment (clairelin135) [inline app/views/works/show.html.slim]: add newline

File: app/javascript/components/detailed_work.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+   
+    return (
+      <div>
+        Button will go here
+      </div>
+    );
+  }
+}
+
+export default DetailedWork;
```
Comment (clairelin135) [inline app/javascript/components/detailed_work.js]: newline

File: db/schema.rb [hunk 5]
```diff
@@ -96,6 +98,7 @@
 
   add_foreign_key "commissions", "artists"
   add_foreign_key "commissions", "buyers"
+  add_foreign_key "requests", "artists"
```
Comment (clairelin135) [inline db/schema.rb]: Makes sense! Didn't think about it this way, it's a good solution

**Pull request merged**